01 - Community/Obsidian Roundup/🗂️ Obsidian Roundup.md

```diff
@@ -11,36 +11,3 @@ publish: true
 The Obsidian Roundup is a weekly newsletter written by written by moderator [[eleanorkonik|Eleanor Konik]]. It is intended to help Obsidian users stay in the loop about community events, code updates and workflow tips. Each issue represents a summary of potentially relevant information the community engaged with in a particular week. You can [subscribe](https://www.obsidianroundup.org/membership/) via email or RSS. 
 
 This archive lags slightly behind the one on the [Roundup's website](https://obsidianroundup.org/) but is interlinked with the rest of the Hub thanks to community efforts. You can contribute to that effort [here](https://github.com/obsidian-community/obsidian-hub/contribute).
-
-## MOC
-
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
--  [[01 - Community/Obsidian Roundup/2021-10-02  Premade Concept Hierarchies & a Virtual Community Meeting Space|2021-10-02  Premade Concept Hierarchies & a Virtual Community Meeting Space]]
```

Comment:
Might make sense to keep this as a manual MOC, not sure. @eleanorkonik thoughts?